lib/menu.rb

```diff
@@ -0,0 +1,35 @@
+class Menu
+  def initialize
+    @menu = {}
```

Comment:
Avoid setting an instance variable with the same name as the class.

`@items` would make more sense here, or `@item_dictionary` if you want to be super clear that you have prices in there too.